lib/controllers/project.js

```diff
@@ -106,6 +106,17 @@ Project.prototype.status = function(projectId, callback) {
 };
 
 //-----------------------------------------------------------------------------
+Project.prototype.resize = function(projectId, servoSize, callback) {
+  librarian.project.update( { id: projectId, servoSize: servoSize }, userConfig.data.apiKey, function(err, status) {
+    if (err) {
+      return callback(err, null);
```

Comment:
I see what you're referring to. The CLI has a lot of things that could have been better. Since there will never be a second parameter to check, there's no need to pass `null` as the second argument. For linting purposes, it would fail. The only time we want to pass `null` as the second parameter is if we require a third parameter to be sent.